.github/workflows/ci-backend.yml

```diff
@@ -264,21 +263,17 @@ jobs:
                   group: ${{ matrix.group }}
                   person-on-events: ${{ matrix.person-on-events }}
 
-            - uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
-              id: add-and-commit
-              if: ${{ matrix.clickhouse-server-image == 'clickhouse/clickhouse-server:22.3' && !matrix.person-on-events }}
+            - uses: EndBug/add-and-commit@v9
+              # Skip on forks
+              # Also skip in extra variant runs (FOSS or persons-on-events), as we want to ignore snapshots diverging there
+              if: ${{ !matrix.person-on-events && !matrix.foss && github.event.pull_request.head.repo.full_name == github.repository }}
               with:
-                  # The arguments for the `git add` command (see the paragraph below for more info)
-                  # Default: '.'
                   add: '["ee/clickhouse", "posthog/api/test/__snapshots__", "posthog/test/__snapshots__", "posthog/queries/"]'
-
-                  default_author: github_actions
-
-                  # The message for the commit.
-                  # Default: 'Commit from GitHub Actions (name of the workflow)'
                   message: 'Update snapshots'
+
             - name: Check if any snapshot changes were left uncomitted
               id: changed-files
+              if: ${{ !matrix.person-on-events && !matrix.foss && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Seems we don't need any check here, as the step is harmless?

Reply:
Right, it shouldn't do anything, but I think it's a bit cleaner to skip it if unneeded